core/src/main/java/org/elasticsearch/index/mapper/geo/GeoPointFieldMapper.java

```diff
@@ -93,10 +96,10 @@
         public static final boolean ENABLE_GEOHASH = false;
         public static final boolean ENABLE_GEOHASH_PREFIX = false;
         public static final int GEO_HASH_PRECISION = GeoHashUtils.PRECISION;
-        public static final boolean NORMALIZE_LAT = true;
-        public static final boolean NORMALIZE_LON = true;
-        public static final boolean VALIDATE_LAT = true;
-        public static final boolean VALIDATE_LON = true;
+
+        public static final boolean BWC = false;
+        public static final Explicit<Boolean> IGNORE_MALFORMED = new Explicit<>(false, false);
```

Comment:
boolean would work fine. For consistency I was following the other mappers  (I wasn't sure if there was some plan for these explicit options?)